python/pywatchman/pybser.py

```diff
@@ -69,6 +69,7 @@ def tobytes(i):
 # our overall length.  To make things simpler, we'll use an
 # int32 for the header
 EMPTY_HEADER = b"\x00\x01\x05\x00\x00\x00\x00"
+EMPTY_HEADER_V2 = b"\x00\x02\x05\x00\x00\x00\x00"
```

Comment:
As a general comment, where are you passing the BSER capabilities down? You probably need another int for that, right? So this should be something more like `"\x00\x02\x05\x00\x00\x00\x00\x05\x00\x00\x00\x00"`, where the first sequence is the capabilities and the second one is the PDU length (or the other way round).